src/components/App.js

```diff
@@ -3,33 +3,34 @@ import {
   Route,
   Routes,
 } from 'react-router-dom';
-
 import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-
-import { getMissions } from '../redux/missions/missionsSlice';
 import MyProfile from './MyProfile';
 import NotMatch from './NotMatch';
 import Layout from './Layout';
 import Missions from './Missions';
 import Rockets from './Rockets';
+import { fetchRockets } from '../redux/rockets/rocketsSlice';
 
-const App = () => {
+function App() {
   const dispatch = useDispatch();
+
   useEffect(() => {
-    dispatch(getMissions());
+    dispatch(fetchRockets());
   }, [dispatch]);
```

Comment:
I noticed that you removed lines 19-21 of my part of the code.
Could you please move them back as it's an important part of my code 🙂 
You can place it before or after your dispatch.